server/certstore/certstore_windows.go

```diff
@@ -112,7 +112,10 @@ var (
 	}
 
 	// MY is well-known system store on Windows that holds personal certificates
-	winMyStore = winWide("MY")
+	winMyStore             = winWide("MY")
+	winIntermediateCAStore = winWide("CA")
+	winRootStore           = winWide("Root")
+	winAuthRootStore       = winWide("AuthRoot")
```

Comment:
For posterity, are there any further docs that can be linked to in a comment in case we need to dig into understanding the semantics of these various stores in the future?

Reply:
I'll add links to the MSFT docs.